data/strings/strings.txt

```diff
@@ -20822,6 +21296,7 @@
     ja = 中間地点を追加
     ko = 스톱 추가
     lt = Pridėti sustojimą
+    lv = Pievienot pieturu
```

Comment:
"Pievienot starppunktu".

Because "Starppunkts" is used everywhere else

Reply:
I agree to the change. (but that means Lithuanian is wrong as well).

Reply:
Dear brother Lithuanians, @organicmaps/translations-lt , are you sure that the word stop (as in "bus stop") is the best choice in "Pridėti sustojimą"? In Latvian we translated as smth like "middle-location".